### ISSUE TITLE
Possible workaround to the lack of Cinnamon Transparency? (WIP)

### ISSUE THREAD

**Issue by feren** [cinnamoner/Clipboard#2]
Basically, I followed these instructions on here: http://askubuntu.com/a/784870/337847
...and the results I got are interesting. Basically, when I logged back in, the titlebars of user-level applications weren't decorated normally anymore, instead, they looked like they were decorated under GNOME!

The good:
- With a bit more work, you could possibly get transparent titlebars!
The ugly:
- Some GNOME Apps won't work after doing this trick, including Software.

**Comment by feren** [comment]
@cinnamoner

**Comment by cinnamoner** [comment]
@feren Interesting.

**Comment by cinnamoner** [comment]
@feren What about the transparent panel thing? Ain't it good for the job? Asked the guy and he said it does the job. I'll link him here asap.

**Comment by cinnamoner** [comment]
Ugh! I've got the vison but nobody is listening. Darn.

**Comment by cinnamoner** [comment]
@feren There. I had a conversation with the guy https://github.com/germanfr/cinnamon-transparent-panels/issues/7

**Comment by cinnamoner** [comment]
@Elbullazul IMPORTANT 

I've got a great vision. You and others have the knowledge. Please tell me once n for all if you want to hear me out. I'm literally sitting on a chair with an open browser and I don't know whether to download or not Linux Mint Cinnamon 18.1. No hurry. Just think it over. I have some ideas to make the Vista theme truly grandiose. Do you want to hear me out or not? Won't be piece of cake but it will be unprecedented. The choice is yours. No pressure at all. If not interested than that's that.

**Comment by Elbullazul** [comment]
@feren move to gnome if you want headerbars, otherwise metacity intercepts WM calls and draws borders.

gnome just redirects WM calls to SSD and then there's no problem

**Comment by cinnamoner** [comment]
Great. At least can you adjust a win theme for the menu I've just forked?

**Comment by Elbullazul** [comment]
@cinnamoner possible, should test it first

**Comment by cinnamoner** [comment]
@Elbullazul I've just fork the eightish/tenish ukui themes. It's css. That should make it easier on you. Think + the 10 icon pack and it will look great, huh? Man I hope I haven't upset those people. Forking is not a rude think. Means I like  their work. Change of plans ... not Downloading Cinnamon. I'm downloading  Martin's thingy. Or who the hack knows. Linux. Hahaha!

**Comment by cinnamoner** [comment]
And it will not glow, huh? That's the best part. Hahaha.    Hey are you sure it's not going to glow? 😄

**Comment by Elbullazul** [comment]
@cinnamoner screenie of uki please, I can't access the website

**Comment by cinnamoner** [comment]
@Elbullazul Hey, Ubuntu Mate might = Windows 8 done right. Hahaha. Where's Balmer to see this?

**Comment by cinnamoner** [comment]
Combing soon ... might not be today though. @bluedxca93 After downloading masterzip, how do I install the thing on Mate? Have no idea.

**Comment by cinnamoner** [comment]
@feren Fun fact: when I watched Dave's video on UKUI, I thought it was made in the United Kingdom (UK) Hahaha 😄😄😄

**Comment by Elbullazul** [comment]
just saw the thing on youtube. seems pretty ok, but should test it to theme it

**Comment by cinnamoner** [comment]
It ain't good 'ol gnomenu from Gnome 2 but it's the next best thing ... might modify it ... it's in C, or something or js. If it's in C, I'll use VS. Hahaha. Ain't it ironically funny?

**Comment by cinnamoner** [comment]
Okay.

**Comment by bluedxca93** [comment]
@cinnanoner.
this hack just forces first headerbar to be a titlebar if menubar is
not defined before.

**Comment by cinnamoner** [comment]
@Elbullazul Hey, I like Gnome. But Gnome 2. Mate is a fork of Gnome 2, right? Plasma and Mate, pretty close to one another, huh? Hahaha 😄😄😄

**Comment by bluedxca93** [comment]
not fully anymore.since mate is now gtk3.

**Comment by cinnamoner** [comment]
> this hack just forces first headerbar to be a titlebar if menubar is
not defined before.

@bluedxca93 Thanks. How do I install the UKUI menu on Ubuntu Mate LTS (now that the two are fully compatible)?

1. Download masterzip 
2. I have no idea
3. Diem
4. 💩

**Comment by bluedxca93** [comment]
sudo python setup.py install in extracted masterzip?

**Comment by cinnamoner** [comment]
@bluedxca93 Yeah, just look at it again and it's in Phyton (I don't know any Phyton so I'll leave it as is; I thought it was C but no).

So@bluedxca93 

sudo ... I'm afraid you'll have to be clearer

sudo ... all commands exactly. Assuming the zip is in the downloads folder. I'm just a stupid Windows guy.

**Comment by cinnamoner** [comment]
sudo apt-get kaboom no luck

**Comment by cinnamoner** [comment]
Can I do it manually as with Stark or something? No sudo.

**Comment by cinnamoner** [comment]
Oh, sudo phyton setup.ph install *name of extracted*. Aha.

**Comment by cinnamoner** [comment]
Wellcome back to Switched to Linux.

**Comment by cinnamoner** [comment]
@Elbullazul the secret of the teal bar (Vista) is in one of Petruci's themes. You just have to look at it.

**Comment by bluedxca93** [comment]
@cinnamoner setup.py . well i have problems with python too. my
knowledge about python is very limited.prefer myself bash and lua over
python.

**Comment by cinnamoner** [comment]
@bluedxca93 Forget Phyton. You are good as you are. One can't know them all.

**Comment by cinnamoner** [comment]
Thought about using a Docker Windows img

**Comment by cinnamoner** [comment]
@feren Change the *An OS made for everyone* slogan and target an audience. It's got too much going. Other than that, there's too much white but the logo is very good. And forget about Zorin. I like the don't break my system.. blah done like in Windows. Other than that, settle for 1 menu. Either Sling or Stark. Not both. Share your thoughts. I'm looking at it with the eyes of a guy comming from Windows. The themes can be improved further but ... hey. Nobody wants to listen.

**Comment by feren** [comment]
@cinnamoner 1. It's good to have choice when it comes to Menus
2. StarkMenu doesn't suit feren's Theme 100%, neither does Slingshot 100% suit Windows themes...

**Comment by cinnamoner** [comment]
@feren Too much choice is detrimental. Let me explain. I've seen all the videos on Feren OS and the reactions are generally positive. But when the bottom line is drawn, they say it's a very good distro but they  still won't use it. And I've been wondering for a long time why not. Here's my conclusion:
1. Too much software. 
2. Too much theming 
3. Too much everything 
***
Choice is good but let the people choose. By that I mean: there's plenty of software in the repo. Let the choose what they want. Any distro exceeding 1.5 is bloated. It's overwhelming. And it's confusing.
***
Almost each time you tell them like hey, "you haven't seen the themer". They change themes by going into settings.
***
So, what do you think? It just crossed my mind and I thought hey, let's tell feren what I think. It doesn't necessarily mean that I'm right.

**Comment by cinnamoner** [comment]
Oh and the themes could be better but nobody wants to listen to me. That menu in the Vista theme for example is still not good enough. Vista's menu doesn't look like that. No way. What's even more annoying is: nobody wants to listen. I have ideas to make it better but nobody wants to listen. Won't insist much longer. CinnXP. Now that one is 99. (9). Because the guy only does XP. It's a refined theme.

**Comment by cinnamoner** [comment]
@feren 
Pick 2 ideas and insist on those. Is it a theme distro? Is it a social app distro? Is it Mac? Is it ... what is it?
What defines the disto? *It's everything for everyone* is not a good answer.

**Comment by Elbullazul** [comment]
@cinnamoner you know the amount of work needed to tweak everything to look like the original? Developers have other things to do beside tweaking the UI to make it look 'like the real thing'. Plus, like raymerjacque said, people don't want windows clones, they want linux to look like windows but not a complete clone. Look at the pseudo-windows linux distros that failed (ylmfos, puppy8, longhorn TP distro, etc.)

and also, please stop whining and instead post valuable feedback on your 'desired changes', 'cause having to read 10 posts to find out you want a better entry isn't fun or interesting

**Comment by Elbullazul** [comment]
> CinnXP. Now that one is 99. (9). Because the guy only does XP. It's a refined theme.

and also because he uses the most complete and stable codebase ever

**Comment by cinnamoner** [comment]
@feren Here's an example. 
- Hey, Ubuntu Kylin, what defines you? 
- I'm a distro aiming to be familiar to the end user by bringing a Windows 7-like layout and a flat theme reminiscent of Windows 8. I'm simple, straightforward, easy to use, easy to understand. And I'm providing you with a software centre to pull in what you need or like.
***

**Comment by cinnamoner** [comment]
@Elbullazul Amen.

**Comment by cinnamoner** [comment]
> you know the amount of work needed to tweak everything to look like the original?

@Elbullazul Nobody forced you to make all those themes (and even less so for virtually all the DEs on this planet). Make Vista, 7, 10. But make them all the way. And cut down on the number of DEs.

> Plus, like raymerjacque said, people don't want windows clones

@Elbullazul You can't make clones even if you wanted to due to the constraints imposed by the desktop environments.

So, neither of the arguments really hold any water. I'm willing to help you. But I can't force you to do what you don't want to. Whatever the reason.

**Comment by cinnamoner** [comment]
@Elbullazul Valuable feedback? My valuable feedback is this: look at the original. Or perhaps you want me to post side by side screenshots for comparison?

**Comment by cinnamoner** [comment]
> CinnXP. Now that one is 99. (9). Because the guy only does XP. It's a refined theme.

> and also because he uses the most complete and stable codebase ever

@Elbullazul Agreed. Do the same. It's the right way of making things right.

**Comment by cinnamoner** [comment]
@Elbullazul When you go public with anything, you can expect criticism.

**Comment by Elbullazul** [comment]
@cinnamoner yes but that 'nobody listens' phrase can't be more false. any thread you remember posting me in where I didn't answer the question you asked?

**Comment by cinnamoner** [comment]
Here's the deal: both of us are only humans. Let us calm down a bit. I do think you know the menu in Vista is not quite there yet as well as a number of other things. Improvement is achievable and I had been hinting at it for a long time. So long in fact that both of us might have lost appetite for that theme altogether. If so, it makes no sense for me to continue talk about it. For the most part ... man I really don't know how to lay it out anymore. Do you take suggestions to make improvements or you've lost interest? If you've lost interest or for whatever reason don't feel like it, I can't blame you and I can understand you.

**Comment by cinnamoner** [comment]
@feren @Elbullazul 
I'm sorry my friends. That's my take. It's not necessary the correct one. Persue whatever makes you happy.
